source/new-component.js

```diff
@@ -14,63 +13,70 @@ const readFile = require('./utils/read-file');
 const renameTransform = require('./transforms/rename-jsx');
 const writeFile = require('./utils/write-file');
 
-module.exports = async function(
-  maybePathOrName,
-  maybeShouldBeStateful,
-  componentsPath
-) {
-  const { prettierConfig } = await getConfigs();
-  const { pathOrName, shouldBeStateful } = await prompt({
-    pathOrName: {
-      text: 'Name of component',
-      value: maybePathOrName
-    },
-    shouldBeStateful: {
-      text: 'Should component have state?',
-      type: Boolean,
-      value: maybeShouldBeStateful
+module.exports = function({
+  componentsPath,
+  eslintConfig,
+  pathOrName,
+  shouldBeStateful
+}) {
+  return new Promise(async (resolve, reject) => {
+    if (!componentsPath) {
```

Comment:
Use `ow` for this? (https://github.com/sindresorhus/ow)

Reply:
This is the case for everywhere we use a check and reject pattern.

Reply:
Do we _really_ need to add a 240kb dependency to save ~15 lines?